pkg/local_object_storage/engine/put.go

```diff
@@ -232,3 +238,70 @@ func (e *StorageEngine) putToShardWithDeadLine(sh shardWrapper, ind int, pool ut
 		}
 	}
 }
+
+// broadcastObject stores object (lock or tombstone) on ALL shards to ensure it's available everywhere.
+func (e *StorageEngine) broadcastObject(obj *objectSDK.Object, objBin []byte, objType objectSDK.Type) error {
```

Comment:
getting `objType` from `obj` can prevent potential mistakes. We use it in text only, does not worth to be a prm